src/utils.rs

```diff
@@ -62,6 +62,13 @@ macro_rules! regex {
     }};
 }
 
+#[macro_export]
+macro_rules! long_version {
+    () => {
+        concat!(env!("CARGO_PKG_VERSION"), "\n", env!("XH_FEATURES"))
+    };
+}
+
```

Comment:
This could have been a normal function instead of a macro rule. However, I am in favour of inlining this logic since it is only used once.

Reply:
Fixed.